platform/darwin/src/MGLAnnotation.h

```diff
@@ -36,6 +41,14 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, readonly, copy, nullable) NSString *subtitle;
 
+#if TARGET_OS_IPHONE
+
+@property (nonatomic) BOOL canShowCallout;
```

Comment:
That's a good point. Maybe we should put them in a separate protocol, MGLCalloutSource, that both MGLPointAnnotation and MGLAnnotationImage could conform to.